services/web/server/src/simcore_service_webserver/email/_core.py

```diff
@@ -35,7 +35,9 @@ def _create_smtp_client(settings: SMTPSettings) -> aiosmtplib.SMTP:
 async def _do_send_mail(
     *, message: MIMEText | MIMEMultipart, settings: SMTPSettings
 ) -> None:
+    #
     # WARNING: _do_send_mail is mocked so be careful when changing the signature or name !!
+    #
 
```

Comment:
...

Reply:
hmm maybe transform that into a docstring... then it will be visible when you hover it as well? or does it work like so as well?